x-pack/plugins/infra/public/components/logging/log_text_stream/log_entry_row.tsx

```diff
@@ -95,6 +99,16 @@ export const LogEntryRow = memo(
     const hasActionViewLogInContext = hasContext && openViewLogInContext !== undefined;
     const hasActionsMenu = hasActionFlyoutWithItem || hasActionViewLogInContext;
 
+    const uiActions = useKibanaContextForPlugin().services.uiActions;
+
+    const externalContextMenuItems = useAsync(() => {
+      return getContextMenuItemsFromActions({
+        uiActions,
+        triggerId: ObservabilityTriggerId.LogEntryContextMenu,
+        context: logEntry,
+      });
+    }, []);
```

Comment:
I'm not too familiar with `useAsync`, but should the `logEntry` be part of the dependency list here?

Reply:
yes - shame that there's no eslint rule for that. Thanks!